docs/internals/service-tokens.mdx

```diff
@@ -5,10 +5,13 @@ description: "Understanding service tokens and their best practices"
 ​
 Many clients use service tokens to authenticate and read/write secrets from/to Infisical; they can be created in your project settings.
 
+On this page, we discuss Service Token V3, the new and improved authentication method.
```

Comment:
same note for here, let's hide v3 docs for now until the feature is made visible to users (which will be when all the clients are compatible with v3 tokens)